lua/pac3/editor/client/parts.lua

```diff
@@ -2673,7 +2716,7 @@ function pace.UltraCleanup(obj)
 			if IsSafe(part) or Important(part) then
 				return
 			elseif IsMildlyRisky(part) then
-				if table.Count(part:GetChildren()) == 0 then
+				if #part:GetChildren() == 0 then
```

Comment:
Are we sure these table.Count replacements will always work? vgui can be pretty inconsistent with the type of table this returns iirc.

Reply:
Maybe use table.Empty instead

Reply:
In theory part:GetChildren() should always return a numerically indexed table. I'm not sure what's up with this code, but I guess in theory is just in theory..

Also I believe [1] is faster than getting the tables length, which part:HasChildren() should do.